src/js/stores/MarathonStore.js

```diff
@@ -159,6 +159,30 @@ var MarathonStore = Store.createStore({
     return this.get('apps')[name];
   },
 
+  getServiceNameFromTaskID: function (taskID) {
+    let serviceName = taskID.split('.')[0].split('_');
+    return serviceName[serviceName.length - 1];
+  },
+
+  getServiceFromTaskID: function (taskID) {
+    let service = this.get('apps')[this.getServiceNameFromTaskID(taskID)];
+    if (service == null) {
+      return new Service();
```

Comment:
You're right. This should return 'null', otherwise it's a too confusing behavior.